contents/feedback/empty-status/simple/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Plus } from "@yamada-ui/lucide"
+import { Text, Button, Flex, Wrap, Icon } from "@yamada-ui/react"
+import type { FC } from "react"
+import { FaRegCreditCard } from "react-icons/fa6"
+
+const Simple: FC = () => {
+  return (
+    <Flex direction="column" justifyContent="center" alignItems="center" h="md">
+      <Icon as={FaRegCreditCard} fontSize="80px" marginLeft="3px" />
```

Comment:
As far as possible, do not use fixed values such as `px`, but set a value close to the token.

https://yamada-ui.com/styled-system/theming/default-theme#spaces
https://yamada-ui.com/styled-system/theming/default-theme#sizes
https://yamada-ui.com/styled-system/theming/default-theme#fontsizes

Reply:
[63cc462](https://github.com/yamada-ui/yamada-components/pull/448/commits/63cc462cc49c72ed8e3b9265ce2ab0e95b0e3bee)